removalScript.js

```diff
@@ -3,9 +3,15 @@ console.log("exectuing...")
 if(window.location.href.split("//")[1].split("/")[0] == "www.youtube.com") {
 	if(window.location.href.split("?")[0].split("/")[3] == "watch") {
 		let items = document.getElementById("items")
-		items.parentNode.removeChild(items)
+		console.log(items)
+		if(items != undefined) {
+			console.log("attempting removal")
+			items.remove()
+		}

```

Comment:
The watch page is now sort of broken? It executes all the code without error and then nothing happens. Except 10% of the time something does happen.

Reply:
Never mind it works 100% of the time for me now??

Reply:
It seems to be function now. The time it takes to kick in is highly inconsistent.